lib/Controllers/list.js

```diff
@@ -106,6 +102,15 @@ List.prototype.fetch = function(req, res, context) {
       var end = start + result.rows.length - 1;
       end = end === -1 ? 0 : end;
 
+      if (options.include) {
```

Comment:
do you think there's any merit to moving this to something like ```before.send```? Like maybe somebody would want to muck with the complete dataset before it gets serialized to JSON? Just sort of playing devils advocate for a second, but otherwise LGTM